modules/ROOT/pages/dw-multipart-functions-field.adoc

```diff
@@ -21,7 +21,7 @@ that use the parameter names as keys, for example:
 [%header, cols="1,3"]
 |===
 | Name | Description
-| `opts` a| Array of objects that specifies:
+| `opts` a | Array of objects that specifies:
```

Comment:
TODO: CHECK if space after "a" messes up the table; probably does. Better to have table directive `[%header, cols="1,3a"]`, with the `a` here instead of in the table itself.